sandbox/mocks/Minimal_Patient.json

```diff
@@ -1,35 +0,0 @@
-{
```

Comment:
This is used as a example response for a minimal patient -> see `src/services/patients.js`

Reply:
Added back on